service/templates/projects.html

```diff
@@ -140,6 +140,17 @@ <h2> Settings for [[displayName]]</h2>
         <input class="form-control" type="text" ng-model="settingsFormData.display_name">
       </div>
     </div>
+    <div class="form-group">
+      <div class="col-lg-2 col-header">
+        <label class="control-label"> Identifier</label>
+      </div>
+      <div class="col-lg-4">
+        <input class="form-control" type="text" ng-model="settingsFormData.identifier">
+      </div>
+    </div>
+    <div class="muted small" style="padding-bottom:12px;width:60%;">
+      Please be aware that changing identifier will change the project identifier used for the CLI, inter-project dependencies, etc.
+      You will need to make your users aware of this change, as using the old identifier will no longer work.</div>
```

Comment:
Does it make sense to change it to `You will need to make any users of this project aware of this change`?